Owncloud iOs Client/Files/Preview/DetailView/DetailViewController.m

```diff
@@ -1747,11 +1736,7 @@ - (void) customWillRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterf
     
     if (_openWith) {
         
-        if (k_use_open_with_UIDocumentInteractionController) {
-            [_openWith.documentInteractionController dismissMenuAnimated:NO];
-        } else {
-            [_openWith.activityPopoverController dismissPopoverAnimated:NO];
-        }
+        [_openWith.documentInteractionController dismissMenuAnimated:NO];
```

Comment:
You can check here not only that **_openWith** exists but also that **_openWith.documentInteractionController** exists.

Reply:
👍